pkg/scheduler/core/generic_scheduler.go

```diff
@@ -112,6 +139,26 @@ type genericScheduler struct {
 	pdbLister                algorithm.PDBLister
 	disablePreemption        bool
 	percentageOfNodesToScore int32
+	// Enable equivalence class cache
+	enableEquivalenceClassCache bool
+
+	nodeInformer                  coreinformers.NodeInformer
```

Comment:
It's not clear to me that all of these informers need to be in the struct. Which ones do we need after `NewGenericScheduler()` has finished?

Reply:
I am trying to reuse the informers passed from cmd/kube-scheduler via NewConfigFactory. I think they are not used after the NewGenericScheduler finishes. Is it ok to not use the ones passed from cmd/kube-scheduler and just a local copy to add the event handlers ?

Reply:
The expectation is that `NewGenericScheduler()` takes informer args for each kind that it cares about. If we add event handlers to those informers, then we might not need to keep a reference to the informer itself.